pages/common/tokei.md

```diff
@@ -0,0 +1,19 @@
+# tokei
+
+> A program that allows you to count your code, quickly.
+
+- Get a report on the code in a folder and all subfolders:
+
+`tokei {{path/to/folder}}`
+
+- Get a report of a folder but ignoring .min.js files:
```

Comment:
Perhaps this will read better?

```suggestion
- Get a report for a folder but ignoring `.min.js` files:
```

Reply:
@sbrl, should `.min.js` have been in double quotes as backticks are usually for commands?